lisp/cues/media_cue.py

```diff
@@ -69,6 +69,11 @@ def __start__(self, fade=False):
         if fade:
             self._on_start_fade()
 
+        # If there was a fade-down, but there wasn't a fade-up - either
+        # because the fade-up was disabled, or because it had a duration
+        # of zero - restore the volume.
+        self.__volume.current_volume = self.__volume.volume
```

Comment:
This should be in an else branch of the previous `if` statement (line 69).
Since `_on_start_fade()` is asynchronous, even when called, this line will get executed immediately after, this can cause some problem that might be unnoticeable when going from `stop` to `running`, due to some delay in loading the file, you can notice it when going from `pause` to `running` (with a fade).